crates/edr_evm/src/mempool.rs

```diff
@@ -337,15 +337,18 @@ impl MemPool {
         ) -> bool {
             transaction.gas_limit() <= block_gas_limit
                 && transaction.upfront_cost() <= sender.balance
+                // Remove all finalized transactions
```

Comment:
Is this comment correct?

Reply:
Maybe the word `finalized` is overloaded?

What I'm trying to convey is that any pending blocks that have been mined should be removed. I'll use the word `mined` instead, as that's more accurate. Let me know if you have a better alternative; thanks!